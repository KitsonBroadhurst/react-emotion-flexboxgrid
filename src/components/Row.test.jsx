/* eslint-env jest */
/* eslint-disable no-unused-expressions */

import React from 'react';
import { shallow } from 'enzyme';

import Row from './Row';

describe('Row', () => {
  it('should work', () => {
    expect(<Row />).toBeTruthy();
    expect(shallow(<Row />)).toMatchSnapshot();
  });
});
