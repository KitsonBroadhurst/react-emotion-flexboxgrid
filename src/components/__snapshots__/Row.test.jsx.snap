// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should work 1`] = `
.emotion-0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

<div
  className="emotion-0 emotion-1"
/>
`;
